ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java

```diff
@@ -70,4 +70,8 @@ public static AdConfig adConfigWithNetworkExtras(Bundle networkExtras) {
         }
         return adConfig;
     }
+
+    public static boolean isStartMuteConfigured(Bundle networkExtras) {
+        return networkExtras.containsKey(EXTRA_SOUND_ENABLED);
```

Comment:
Can we rename this to be EXTRA_START_MUTED as elsewhere we have similar name and would be easy to match that value elsewhere and up to the SDK.
Also should we accept a boolean value for this key/property (true/false), so that for any specific scenario its explicit and Publisher can set the behavior accordingly.

Reply:
fixed.